test/controllers/api/v2/ostree_branches_controller_test.rb

```diff
@@ -67,10 +67,21 @@ def test_compare
 
     def test_branch_order
       @repo.ostree_branches.create!(:name => "def456", :uuid => "456uvw", :version => "1.0")
+      @repo.ostree_branches.create!(:name => "ghi789", :uuid => "789rst", :version => "1.11")
+      @repo.ostree_branches.create!(:name => "jkl123", :uuid => "123opq", :version => "1.10")
+      @repo.ostree_branches.create!(:name => "mno456", :uuid => "456lmn", :version => "1.11.12")
+      @repo.ostree_branches.create!(:name => "pqr789", :uuid => "789ijk", :version => "1.11.120")
+      @repo.ostree_branches.create!(:name => "stu123", :uuid => "123fgh", :version => "1.11.119")
```

Comment:
The bug has some example versions like '7.3.2-1' which you'll want to account for as well I think